VespaGAS/Managers/Reducer/MainReducer.swift

```diff
@@ -0,0 +1,9 @@
+import ReSwift
```

Comment:
Reducers are the only one that can modify the state of your app. 
Here we have a main one for the entire app, each component of the state has its own reducer, for example oilMixState has the oilMixReducer. 

Reducer are pure functions, they return the new state depending on the actual state + action provided.